### DISCUSSION TITLE
Disabling `autoActivate` via CLI flags

### DISCUSSION THREAD

**Discussion by talss89** [abiosoft/colima#954] [Q&A] [answered]
Hello,

I might have missed something, but is there a way to disable `autoActivate` via CLI flags?

Looking over the source, `--activate` seems to always be `true`, seemingly for legacy support.

I would like to preserve the existing `DOCKER_CONTEXT` after running `colima start`, as I'm using Colima as a dependency for a CLI tool I'm working on.

Thanks for any help!

Tom.

**Comment by abiosoft** [comment, marked as answer]
`--activate=false`

**Reply by talss89** [reply]
Not sure why I thought this didn't work. It does, thank you.
